spec/models/user_spec.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_mock) { build(:user) }
```

Comment:
I'd just say `user` and `user_two`, no need for `mock`. In testing, a `mock` is something specific and we're not mocking here.

Reply:
Some additional reading as well: https://relishapp.com/rspec/rspec-mocks/docs